pliers/extractors/api/awsReko_test.py

```diff
@@ -0,0 +1,82 @@
+'''
+Extractors that interact with the AWS Rekognition API.
+'''
+
+from pliers.stimuli.image import ImageStim
+from pliers.extractors.base import Extractor, ExtractorResult
+import numpy as np
+import pandas as pd
+import boto3
+from pliers.utils import attempt_to_import, verify_dependencies
+
+aws_rekognition_client= attempt_to_import('boto3')
+
+
+class AwsRekognitionExtractor():
+    
+    def __init__(self,profile_name='default', region_name='us-east-1' ):
+            verify_dependencies(['boto3'])
+            self.session = boto3.Session(profile_name='user2')
+            self.rekognition = boto3.Session.client('rekognition', region_name='us-east-1')
+            super(AwsRekognitionExtractor, self).__init__()
+    
+    
+    def _extract(self, stim):
+        img2byte=self.img_to_byte(stim)
+        response=self.rekognition.detect_faces(
+            Image={
+                'Bytes': img2byte
+                },
+                    
+            Attributes=['ALL']
+        )
+        return response
+    
+    
+    @property
+    def img_to_byte(img):
+    #     Convert image to bytes
+        with img.get_filename() as filename:
+            with open(filename, "rb") as image:
+                f = image.read()
+                b = bytearray(f)
+                return(b)
+                
+   
+    def to_df(self, d,result=None,index=None,Key=None):
```

Comment:
It would be good to get in the habit of running your code against PEP8. You can user the standalone `pylint` package, but it's easier to use an IDE that supports linting plugins. Here the issue is the lack of spacing between arguments.